src/main/java/coydir/model/person/Person.java

```diff
@@ -129,13 +161,24 @@ public String toString() {
                 .append("; Position: ")
                 .append(getPosition())
                 .append("; Address: ")
-                .append(getAddress());
+                .append(getAddress())
+                .append("; Total Leaves: ")
+                .append(getTotalNumberOfLeaves())
+                .append("; Number of Leaves Left: ")
+                .append(getTotalNumberOfLeaves());
```

Comment:
This should be `.append(getLeavesLeft())`?

Reply:
Yes Boss